GOVERNANCE.md

```diff
@@ -1,6 +1,6 @@
 # OpenJS Cross Project Council
 
-For the current list of Team members, see the project [README.md][README].
+For the current list of Team members, see the project [README.md][README] and the [OpenJS Foundation Directory](https://github.com/openjs-foundation/directory-private/blob/master/cpc-private.md).
```

Comment:
I suggest we use as a rule to always mark private information as such and give the reason as to why it is private. 

```suggestion
For the current list of Team members, see the project [README.md][README] and the [OpenJS Foundation Directory](https://github.com/openjs-foundation/directory-private/blob/master/cpc-private.md) (restricted access for privacy reasons).
```

Reply:
This looks good, thanks @tobie.